### DISCUSSION TITLE
Please upgrade your Copilot extension to continue using this service.

### DISCUSSION THREAD

**Discussion by wl9180** [community/community#18785] [Copilot Conversations] [answered]
Please upgrade your Copilot extension to continue using this service.
这个问题咋解决？
1.17版本的github copilot ，机型是macbookpro m1

**Comment by davecheney** [comment, marked as answer]
@wl9180 please upgrade to version 1.29 (or later) of the VSCode extension, or version 1.1.25 (or later) of the Intellij extension.
